src/es/components/pages/Inspirationsdetail.html

```diff
@@ -1,8 +1,15 @@
+<style>
```

Comment:
please comment if this is for preview purposes...

Reply:
Hab jetzt nen Kommentar rein. Das ist der Mechanismus, wie headlines im Inspirationsteil angepasst werden an brand bzw. Thema. Das muss irgendwo im BE auch hinterlegt sein.